internal/flags/apiflags/flags_test.go

```diff
@@ -111,39 +111,9 @@ func TestVerifyCommonFlags(t *testing.T) {
 			operatorType: "go",
 			expError:     "",
 		},
-
-		{
-			// InValid Go API Flags
```

Comment:
Why these tests were removed?

Reply:
@camilamacedo86 These tests were not designed correctly for the function, hence they have been removed.

Reply:
Agreed, resolving this conversation.